backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/entity/Trip.java

```diff
@@ -0,0 +1,29 @@
+package com.chinguv45tier3team53.tripplannerrest.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Entity
+@Builder
+@Table(name="trips")
+public class Trip {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name="user_id",nullable=false)
```

Comment:
the fields linking to other classes will have to change once we have them.
Should be nullable for now, otherwise requests from the client will need to include these fields, and someone have to remember to fix these as well

Reply:
Those are foreign keys.., but sure